README.md

```diff
@@ -6,15 +6,15 @@
 <a href="https://github.com/aave-starknet-project/aave-starknet-bridge/blob/main/LICENSE.md"><img src="https://img.shields.io/github/license/sourcerer-io/hall-of-fame.svg?colorB=ff0000"></a>
 
 
-:warning: This codebase is still in an experimental phase, has not been audited, might contain bugs and should not be used in production.
-
+:warning: This codebase is still in an experimental phase, has not been
+audited, might contain bugs and should not be used in production.
```

Comment:
why a newline? probably a mistake

Reply:
EDIT: those newlines come from a markdown formatter?

Reply:
vim auto wrapping to 80 chars, I'd like to have them in because it makes it a lot easier to work with but I can restore it to the previous state of single lines